src/Bridge/Doctrine/Orm/Filter/OrderFilter.php

```diff
@@ -34,25 +36,9 @@
  * @author Kévin Dunglas <dunglas@gmail.com>
  * @author Théo FIDRY <theo.fidry@gmail.com>
  */
-class OrderFilter extends AbstractContextAwareFilter
+class OrderFilter extends AbstractContextAwareFilter implements OrderFilterInterface
 {
-    const NULLS_SMALLEST = 'nulls_smallest';
-    const NULLS_LARGEST = 'nulls_largest';
-    const NULLS_DIRECTION_MAP = [
-        self::NULLS_SMALLEST => [
-            'ASC' => 'ASC',
-            'DESC' => 'DESC',
-        ],
-        self::NULLS_LARGEST => [
-            'ASC' => 'DESC',
-            'DESC' => 'ASC',
-        ],
-    ];
-
-    /**
-     * @var string Keyword used to retrieve the value
-     */
-    protected $orderParameterName;
```

Comment:
this protected value is now private.

Reply:
Forgot to change it, yes. Did it cause an issue?

Reply:
BC break for my app, we extended `OrderFilter` to use the work on properties and the inheritance of `orderParameterName` of the constructor as it's not a final nor an internal class I think it maybe a BC break (and for no additional value?)

Reply:
In fact, the `OrderFilter` _should_ be final but is not (the MongoDB one is final).
But you're right this is a BC break, could you make a PR to fix it?

Reply:
Sorry, I don't understand why the`OrderFilter` should be final. There are many cases where I've needed to extend it.

Reply:
I've not so much time to make PR right now, feel free @teohhanhui (hey again) or @alanpoulain.

Reply:
https://github.com/api-platform/core/pull/2678